lib/rules/indent.js

```diff
@@ -616,6 +617,7 @@ module.exports = {
             MemberExpression: 1,
             ArrayExpression: 1,
             ObjectExpression: 1,
+            ImportDeclaration: 1,
```

Comment:
It's worth noting that this option will not apply to `ExportNamedDeclaration`:

```js
export { foo,
         bar,
         baz
} from 'qux';
```

Would we also add an option for that? We could alternatively put them under the same option (for example, the `CallExpression` option also applies to `NewExpression` nodes), but I'm not sure if that would be confusing.